src/lib/environment.js

```diff
@@ -9,17 +9,19 @@ const isStaging = NODE_ENV === "staging"
 const isProduction = NODE_ENV === "production"
 const isDeploy = isStaging || isProduction
 const isCI = CI === "true"
+const isServer = typeof window === "undefined"
```

Comment:
seeing this defined in a file called `environment` threw me for a second (i thought this file had to do with 'environment variables', and it seems like there are `process.env` references, etc.)

but i guess it's just helpers related to the environment? im prob totally overthinking this, but what does requiring `ANALYZE_BUNDLE` (for example) from here on the client look like? is that accurate (since that value would be set in the generated JS assets already)?

Reply:
> but i guess it's just helpers related to the environment

Yup, just general helpers. I think its probably ok for this to be contextual, like a dev will know when `ANALYZE_BUNDLE` is needed and where it would be valid, etc. But down to omit this if it feels confusing.